mautic/mautic
# Fix JS translations

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.3
| Bug fix?                               | ✅ 
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | ✅ 
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:

We are able to use translations in js Mautic.translate(). But this does not support english is default language, If translation does not exists in user language.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Open Mautic in default English language
3. Open the browser console and  run `Mautic.translate('mautic.core.builder.theme_change_warning')`
4. You should see string: **You will lose the current content if you switch the theme**
5. Switch from profile language to Indonesia
6. Refresh browser
7. Open the browser console and  run `Mautic.translate('mautic.core.builder.theme_change_warning')`
8. You should see **mautic.core.builder.theme_change_warning**
9. Because that string does not exists in Indonesia language, we expect default english
10. After patch you should see correct string




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/9749"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
